ce/marketing/developer/extend-segments.md

```diff
@@ -211,104 +211,72 @@ With the delete request, you delete the compound segment that is created.
 DELETE {{OrgUrl}}/api/data/v9.0/msdyncrm_segments({{SegmentId}})
 ```
 
-## Add/Remove segment members
+## Add/Remove contacts to static segment
```

Comment:
Typo: to static segment**s**

Reply:
PR was merged before,
I addressed these in https://github.com/MicrosoftDocs/dynamics-365-customer-engagement/pull/1720